### DISCUSSION TITLE
Can't sign up for Vercel

### DISCUSSION THREAD

**Discussion by jalenzz** [vercel/vercel#4722] [Help] [answered]
I try to sue Github to sign up, but it showed a error.

> Error:Sorry, we are unable to complete your signup. Please contact us at support@vercel.com for further assistance.

However, I have seen Vercel in my github Applications.

**Comment by styfle** [comment, marked as answer]
Hi @Royce003 

It might be related to your email address.

If you email support@vercel.com, we'll get it sorted out for you. Thanks!
